markdown/assignments/sheet09/index.md

```diff
@@ -1,26 +1,41 @@
 ---
 type: assignment
-title: "Blatt 09: MLP"
+title: "Blatt 09: Backpropagation"
 author: "Canan Yıldız (Türkisch-Deutsche Universität)"
-hidden: true
+points: 10
+hidden: false
 weight: 9
-sketch: true
 ---
 
 
-Hier kommt der Inhalt für Blatt XYZ hin ... allgemeine einleitende Worte ...
+## A9.1: Backpropagation: Hidden Layer (2P)
 
-## Aufgabe 1: XYZ (2P)
+In der Vorlesung wurde(n) die Gewichtsupdates bei der Backpropagation für die Ausgabeschicht und die davor liegende letzte versteckte Schicht hergeleitet, wobei in der Ausgabeschicht die Sigmoid und in der versteckten Schicht die ReLU Aktivierungsfunktionen eingesetzt wurden.
+Leiten Sie die Gewichtsupdates für die erste versteckte Schicht (für ein Netz mit zwei echten versteckten Schichten) her. Verwenden Sie dabei die Sigmoid Funktion als Aktivierung in allen Schichten.
 
-tbd
+*Thema*: Verständnis Backpropagation
 
-## Aufgabe 2: XYZ (3P)
 
-tbd
+## A9.2: Forward- und Backpropagation (3P)
 
+Betrachten Sie das folgende MLP mit einer versteckten Schicht mit zwei Zellen. Die Gewichte sind an den Kanten angegeben. Das Netz erhält den skalaren Input $x$ und berechnet daraus die Ausgabe $\hat{y}$. Beide Zellen verwenden die Aktivierungsfunktion
+$\sigma(z) = \frac{1}{ 1 + e^{−z} }$.
 
+`{{< img src="images/mlp.png" title="Abbildung 1" width="80%" >}}`{=markdown}
 
-`{{% challenges %}}`{=markdown}
-## Bonus: Wuppie (2P)
-Erstellen Sie ...
-`{{% /challenges %}}`{=markdown}
+*   Berechnen Sie die Ausgabe $\hat{y}$ für die Eingabe $(x,y)=(0, 0.5)$. Wie groß ist der Fehler?
+
+*   Berechnen Sie die partiellen Ableitungen für die Gewichte. Wie lauten die Gewichtsupdates für das obige Trainingsbeispiel? Setzen Sie $\alpha = 0.01$.
+
+
+## A9.3: MLP und Backpropagation (5P)
```

Comment:
@cyildiz Willst Du hier evtl. noch Teilpunkte angeben?

Reply:
Guter Punkt, danke!

Reply:
Hmmm, ich sehe in A9.3 keine Teilpunkte, aber vielleicht ist das hier auch schwierig :)